### ISSUE TITLE
User story - Tech Search

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#39]
### If applicable, add a related Epic.

Epic: #42, #45 

### User Story

As a **user**, I can **search for specific technologies** so that I can **see associated job posts or projects with those technologies**.

### Acceptance Criteria

- The user should have the option to search for multiple tech in a search
- Projects associated with that tech should populate on the tech search page

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [x] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [ ] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [x] Updated the issue with any relevant notes or feedback (to date).
- [x] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] May be required to create a Search app, which will contain all the search functionality
- [x] Research search functionality with Django

**Comment by stephendawsondev** [comment]
After reviewing the user stories, I've moved this to `could have` to fit in higher-priority user stories.

**Comment by stephendawsondev** [comment]
I've labelled #17 , #23  and #35  as `5` because I would be starting with the tech search. That gets a 13 because I have no idea how the search works with Django and it could require some significant time to wrap my head around. However, once I have done it with tech, I am hoping it will be clearer how to do it with Projects, users and job posts. It might be that I can even combine them into one.

**Comment by stephendawsondev** [comment]
This was completed in #96
